.gitignore

```diff
@@ -5,5 +5,6 @@ node_modules
 dist
 
 *.js
+!jest.config.js
```

Comment:
Don't ignore the jest config, the line above would ignore it for being a js file